lib/cc_config.h

```diff
@@ -45,11 +45,11 @@ struct LOG_FLAGS {
 
     // on by default; intended for all users
     //
-    bool file_xfer;
+    bool file_xfer = true;
```

Comment:
Currently we're still using VS2010 for Windows release. This code will not work because this type of initialization was added in c++11 only

Reply:
OK, confirmed. Why didn't somebody say that before? Dug out an ancient copy of Visual Studio Express 2010, and the exact error message is

`only static const integral data members can be initialized within a class`

Was there any real reason for eliminating memset in this particular function? As I read it, the concern was with clearing complex structures? This one is a simple array of boolean values (fixed length), it's initialised at startup, and released when the program exits. Is that dangerous? If not, why don't we revert here?

Reply:
memset isn't initialize the members of the structure, it's just initialize a block of memory. I'll look for better solution on this weekend.

Reply:
The original code is visible in the 7.14 release branch:
```
LOG_FLAGS::LOG_FLAGS() {
    init();
}

void LOG_FLAGS::init() {
    memset(this, 0, sizeof(LOG_FLAGS));
    // on by default (others are off by default)
    //
    task = true;
    file_xfer = true;
    sched_ops = true;
}
```
The memset sets all to zero, and then the three defaults are set separately.
The replacement in master uses the same default value initialisers, but they are ineffectual.